gvsbuild install gobject-introspection: Path not escaped

When I run `gvsbuild install gobject-introspection` in a GitHub-provided image in GitHub actions, I get this error:

```
2023-07-26T22:01:03.4694819Z [1208/1209] Generating po/zh_TW/LC_MESSAGES/glib20-zh_TW.mo with a custom command
2023-07-26T22:01:03.4695315Z [1208/1209] Installing files.
2023-07-26T22:01:06.8246281Z (tar) Exporting gobject-introspection
2023-07-26T22:01:06.8291158Z Building project gobject-introspection (1.76.1)
2023-07-26T22:01:06.8517088Z (Stripping trailing CRs from patch; use --binary to disable.)
2023-07-26T22:01:06.8522600Z patching file giscanner/utils.py
2023-07-26T22:01:07.2446240Z usage: meson [-h]
2023-07-26T22:01:07.2447037Z              {setup,configure,dist,install,introspect,init,test,wrap,subprojects,rewrite,compile,devenv,env2mfile,help}
2023-07-26T22:01:07.2447717Z              ...
2023-07-26T22:01:07.2448263Z meson: error: unrecognized arguments: Files (x86)\pipx\venvs\gvsbuild\Scripts\python.exe
2023-07-26T22:01:07.3617610Z Traceback (most recent call last):
2023-07-26T22:01:07.3619077Z   File "C:\Program Files (x86)\pipx\venvs\gvsbuild\lib\site-packages\gvsbuild\utils\builder.py", line 525, in build
2023-07-26T22:01:07.3619647Z     if self.__build_one(p):
2023-07-26T22:01:07.3620261Z   File "C:\Program Files (x86)\pipx\venvs\gvsbuild\lib\site-packages\gvsbuild\utils\builder.py", line 661, in __build_one
2023-07-26T22:01:07.3620736Z     skip_deps = proj.build()
2023-07-26T22:01:07.3621341Z   File "C:\Program Files (x86)\pipx\venvs\gvsbuild\lib\site-packages\gvsbuild\projects\gobject_introspection.py", line 62, in build
2023-07-26T22:01:07.3621798Z     Meson.build(
2023-07-26T22:01:07.3622354Z   File "C:\Program Files (x86)\pipx\venvs\gvsbuild\lib\site-packages\gvsbuild\utils\base_builders.py", line 49, in build
2023-07-26T22:01:07.3622861Z     self._setup_meson_and_ninja(ninja_build, meson_params, add_path)
2023-07-26T22:01:07.3623529Z   File "C:\Program Files (x86)\pipx\venvs\gvsbuild\lib\site-packages\gvsbuild\utils\base_builders.py", line 81, in _setup_meson_and_ninja
2023-07-26T22:01:07.3624016Z     self.exec_vs(cmd, add_path=add_path)
2023-07-26T22:01:07.3624581Z   File "C:\Program Files (x86)\pipx\venvs\gvsbuild\lib\site-packages\gvsbuild\utils\base_project.py", line 195, in exec_vs
2023-07-26T22:01:07.3625031Z     self.builder.exec_vs(
2023-07-26T22:01:07.3625587Z   File "C:\Program Files (x86)\pipx\venvs\gvsbuild\lib\site-packages\gvsbuild\utils\builder.py", line 935, in exec_vs
2023-07-26T22:01:07.3626016Z     self.__execute(
2023-07-26T22:01:07.3626546Z   File "C:\Program Files (x86)\pipx\venvs\gvsbuild\lib\site-packages\gvsbuild\utils\builder.py", line 1039, in __execute
2023-07-26T22:01:07.3627044Z     subprocess.check_call(args, cwd=working_dir, env=env, shell=True)
2023-07-26T22:01:07.3627528Z   File "C:\hostedtoolcache\windows\Python\3.9.13\x64\lib\subprocess.py", line 373, in check_call
2023-07-26T22:01:07.3627979Z     raise CalledProcessError(retcode, cmd)
2023-07-26T22:01:07.3629300Z subprocess.CalledProcessError: Command '"C:\Program Files (x86)\pipx\venvs\gvsbuild\Scripts\python.exe" C:\gtk-build\tools\meson-1.1.1\meson.py setup C:\gtk-build\build\x64\release\gobject-introspection C:\gtk-build\build\x64\release\gobject-introspection\_gvsbuild-meson --prefix C:\gtk-build\gtk\x64\release --buildtype debugoptimized -Dpython=C:\Program Files (x86)\pipx\venvs\gvsbuild\Scripts\python.exe -Dcairo_libname=cairo-gobject-2.dll' returned non-zero exit status 2.
2023-07-26T22:01:07.3630413Z Error: gobject-introspection build failed
2023-07-26T22:01:07.6034378Z ##[error]Process completed with exit code 1.
```

This seems to be caused by the path to `C:\Program Files (x86)` not being escaped or quoted.

--- a/gvsbuild/projects/gobject_introspection.py
+++ b/gvsbuild/projects/gobject_introspection.py
@@ -61,6 +61,5 @@ class GObjectIntrospection(Tarball, Meson):
 
         Meson.build(
             self,
-            meson_params="-Dpython=%s\\python.exe -Dcairo_libname=cairo-gobject-2.dll"
-            % (py_dir,),
+            meson_params=f'-Dpython="{py_dir}\\python.exe" -Dcairo_libname=cairo-gobject-2.dll',
         )

--- a/gvsbuild/projects/gstreamer.py
+++ b/gvsbuild/projects/gstreamer.py
@@ -117,7 +117,7 @@ class GstPluginsBase(Tarball, Meson):
 
     def build(self):
         Meson.build(
-            self, meson_params=f"-Dc_link_args={self.builder.gtk_dir}\\lib\\ogg.lib"
+            self, meson_params=f'-Dc_link_args="{self.builder.gtk_dir}\\lib\\ogg.lib"'
         )
         self.install(r".\COPYING share\doc\gst-plugins-base")
 

